microsoft-r/operationalize/how-to-create-manage-session-pools.md

```diff
@@ -5,7 +5,7 @@ title: "Create and manage session pools for fast web service connections in R (M
 description: "Allocate resources for pre-loading web service connections and dependencies in R solutions (Machine Learning Server ). "
 keywords: ""
 author: "HeidiSteen"
-ms.author: "heidist"
+achms.author: "heidist"
```

Comment:
can you remove the ach form achms.author?  Metadata is used in internal reports and if there are errors, it will break reporting.